util/cinc-install.sh

```diff
@@ -586,20 +603,6 @@ case $machine in
     ;;
 esac
 
-if test "$platform" = "ubuntu"; then
```

Comment:
I think should be kept, depending on how the deb package is named, e.g. if it named after arm64.deb or amd64.deb